src/fu-engine.c

```diff
@@ -5707,6 +5707,7 @@ fu_engine_update_history_device (FuEngine *self, FuDevice *dev_history, GError *
 		fu_device_set_version (dev_history, fu_device_get_version (dev));
 		fu_device_remove_flag (dev_history, FWUPD_DEVICE_FLAG_NEEDS_ACTIVATION);
 		fu_device_set_update_state (dev_history, FWUPD_UPDATE_STATE_SUCCESS);
+		fu_device_set_update_error (dev_history, NULL);
```

Comment:
Has this been a problem in practice?  I think it does seem like a _logical_ thing to do, but when you're talking about UEFI errors I feel like it's not going to matter.  You'll get success or you won't, not a weird transient during your current boot.

Reply:
One Lenovo update: https://github.com/fwupd/fwupd/issues/2171

Reply:
Nope, I meant https://github.com/fwupd/fwupd/issues/2118 -- sorry.